adapters/zwave/zwave-classifier.js

```diff
@@ -46,7 +47,28 @@ class ZWaveClassifier {
             'name': 'toggle',
             'description': 'Toggles On/Off',
         }];
-        node.propertyMap = {'on': valueId };
+        node.propertyMaps = {'on': valueId };
+        node.values.on = node.zwValues[valueId].value;
+
+        if (node.zwInfo.productId == 0x0060 &&
+            (node.zwInfo.productType & 0xff) == 0x03) {
```

Comment:
I'm grateful that you figured out how to make this work, but I really don't think we should bury vendor-specific hacks in the code like this. I'd suggest we either need generic code which can provide a sensible default configuration for all Z-Wave switches (preferable, but maybe not possible), or a mechanism to list vendor-specific default configurations (maybe in some declarative form).

I think it makes sense to have a classifier which maps standardized Z-Wave device types (groups of command classes) onto Web Thing types, but if this kind of vendor-specific configuration is needed to make Z-Wave smart plugs sensibly then it's likely people are going to want to extend this with their own device configurations.

I'd suggest our focus should be on designing a some kind of add-on mechanism for adapters (and specific device configurations if necessary), rather than hard coding these quirks into the classifier.